src/main/java/be/syntra/devshop/DevshopBack/services/UserServiceImpl.java

```diff
@@ -86,8 +87,14 @@ private Authentication getAuthentication(String userName, String password) {
         return authenticationManagerBuilder.getObject().authenticate(authenticationToken);
     }
 
-    public User getUserById(Long userId) {
-        return userRepository.findById(userId)
-                .orElseThrow(() -> new UserNotFoundException(String.format("User with id %s could not be found", userId)));
+    public User getUserByName(String name) {
+        return userRepository.findUserByEmail(name)
```

Comment:
here you see that it's the email that is used to do the lookup inside a method that says its the lookup is done by name, so var's and methods should be name accordingly otherwise very much confusion